src/reaper_osara.rc

```diff
@@ -63,8 +63,11 @@ BEGIN
 	CONTROL "Report &FX when moving to tracks/takes", ID_CONFIG_REPORT_FX, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 126, 140, 14
 	CONTROL "Report MIDI &notes in MIDI editor", ID_CONFIG_REPORT_NOTES, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 146, 140, 14
 	CONTROL "Report changes made via &control surfaces", ID_CONFIG_REPORT_SURFACE_CHANGES, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 10, 166, 155, 14
-	DEFPUSHBUTTON "OK", IDOK, 10, 186, 30, 14
-	PUSHBUTTON "Cancel", IDCANCEL, 137, 186, 40, 14
+#ifdef _WIN32
```

Comment:
Does the PHP script which generates SWELL resources respect ifdefs? In other words, do we know for certain that this actually excludes the control on Mac?

Reply:
AH that's actually pretty unlikely looking at the script. I wonder how to fix this.

Reply:
I think you'll need to do this at runtime, after the dialog is created but before it is shown. If Mac, get the HWND for the checkbox and hide it. You should be able to use ShowWindow with the right arguments to hide it, though I don't recall the specifics. You could test that on Windows easily enough. Failing that, you could just disable it with EnableWindow(hwnd, false) which I know for a fact works because I do that elsewhere.